### ISSUE TITLE
GH_TOKEN for ArgoCD hooks

### ISSUE THREAD

**Issue by shsingh** [apcj-f5/hapi.f5labs.dev#41]
Is it possible to change this token so its not using your PAT and using the repo or API token used in other scripts? 
<img width="499" alt="Screenshot 2023-06-03 at 10 11 20" src="https://github.com/apcj-f5/hapi.f5labs.dev/assets/412800/59fc1792-4293-46ef-bbcf-1fc5f5fc4c68">

**Comment by leonseng** [comment]
Can't use the repo Github token as it's not being run in GH action. Argocd is calling the Github API () to set the status of the commit, as an external client, so a PAT is required. Perhaps a separate GH "robot" account for all things automation?

**Comment by shsingh** [comment]
of course. makes sense. robot is a good idea but we can drop priority of this for now i think. On 4 Jun 2023, at 16:36, Leon Seng ***@***.***> wrote:﻿
Can't use the repo Github token as it's not being run in GH action. Argocd is calling the Github API to set the status of the commit, as an external client, so a PAT is required. Perhaps a separate GH "robot" account for all things automation?

**Comment by shsingh** [comment]
could we change this to an org based PAT for apcj-f5 @leonseng ? would that work?
